models/zoo.yaml

```diff
@@ -1,2 +1,4 @@
 speechio_kaldi_multicn:
   url: oss://speechio-leaderboard/models/speechio_kaldi_multicn/
+wenet_multi_cn:
```

Comment:
Binbin, I've seen WeNet model in cloud model zoo, everything seems OK.  So here in this PR, an update of models/zoo.yaml should be enough, no need to include model-image files in github repo.